infra/recipes/k8s/environments/azure/sandbox/housetables/values.yaml

```diff
@@ -0,0 +1,34 @@
+namespace: "default"
+
+openhouseClusterTag: "openhouse"
+
+clusterPropertiesPath: "config/azure/sandbox/cluster.yaml"
+
+htsService:
+  replicaCount: 1
+  image:
+    repository: "containerregistryopenhouse.azurecr.io/openhouse-housetables-service"
+    tag: "latest"
+    pullPolicy: IfNotPresent
```

Comment:
Why not use `always`? The image in the acr can change.

Reply:
Check if this is the reason your pod is not pulling the latest image.

Reply:
Okay I will change. I will check in the next PR when I am actually modifying the image base files